### PR TITLE
PRs 5212, 5226, 5232; bug 1410758, and move around string

### PR DESCRIPTION
**Pull request by peiying2**
*PR5212: Fx new landing page - add new strings, delete one string.
*PR5226: a footer string
*PR5232: whatsnew 57. Version B. 
*bug1410758: add string "History"
*download_button.lang: add a string.

### CODE REVIEW

**Comment by peiying2** [comment]
For whatsnew_57, there is no testing URL. I can note that in the communication. Should we remove testing URLs? Or leave it there as the links are pretty standard.

**Comment by peiying2** [comment]
I have not removed the string from firefox/new/quantum. Should we run leverage or should we let Pontoon do the work (machinery)? Also obsolete files contain the same string.

File: en-US/firefox/whatsnew_57.lang [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## NOTE: Demo page at https://www-demo3.allizom.org/firefox/57.0/whatsnew/
```
Comment (flodolo) [inline en-US/firefox/whatsnew_57.lang]: This is not the DEMO server, we need to push craig to have one before we send out files for translation

File: en-US/main.lang [hunk 2]
```diff
@@ -309,14 +309,13 @@ Portions of this content are ©1998–%(current_year)s by individual mozilla.org
...
-# Footer string (will replace "Contribute to this page")
+# Footer string
 ;Contribute to this site
 Contribute to this site
 
 
-# Footer string
-;Contribute to this page
-Contribute to this page
+;Source code for this page
```
Comment (flodolo) [inline en-US/main.lang]: Good call on removing the other string, but I would add a comment, e.g. "Used in footer, links to the source code of the page currently displayed".

**Pull request merged**